src/main/java/frc/team2412/robot/Subsystems.java

```diff
@@ -38,5 +41,8 @@ public Subsystems() {
 		if (INTAKE_ENABLED) {
 			intakeSubsystem = new IntakeSubsystem();
 		}
+		if (INTAKE_ENABLED) {
```

Comment:
Should this be LED_ENABLED?

Reply:
Agreed, looks like the wrong flag to me.

Reply:
yeah oopsie